pkg/apply/applier.go

```diff
@@ -96,13 +88,12 @@ type Applier struct {
 // Initialize sets up the Applier for actually doing an apply against
 // a cluster. This involves validating command line inputs and configuring
 // clients for communicating with the cluster.
-func (a *Applier) Initialize(cmd *cobra.Command, paths []string) error {
-	fileNameFlags, err := common.DemandOneDirectory(paths)
-	if err != nil {
-		return err
+func (a *Applier) Initialize(cmd *cobra.Command) error {
```

Comment:
Eventually, we want to remove the `cobra.Command` parameter, since it couples the `Applier` to the cli, and we want to use the `Applier` as a library.

Reply:
Yeah, after this change we are almost at a point where we no longer need to call `ApplyOptions.Complete`. This means we will no longer have to deal with cobra flags set inside `ApplyOptions` and we might be able to hide our usage of `ApplyOptions` inside the `ApplyTask`.